reversed.go

```diff
@@ -0,0 +1,19 @@
+package do
+
+import "reflect"
+
+/*
+Reversed returns the values of a given interface on a reversed basis.
+*/
+func Reversed(iterable interface{}) interface{} {
+	iterValue := getIterable(iterable)
+	returnValue := reflect.MakeSlice(iterValue.Type(), iterValue.Len(), iterValue.Len())
+
+	j := 0
```

Comment:
it should be returnValue.Index(iterLen - 1 - i).Set(iterValue.Index(i))
otherwise it will run out of bounds